doc/source/getting-started/administration.rst

```diff
@@ -38,47 +39,46 @@ The approval process is divided into three parts:
     Once approved, a project can be published to the :ref:`Public PyPI`.
 
 
-When releasing a project to the public, expect to:
+When releasing a project to the public, you should:
 
 * Coordinate with the product line development team, if applicable.
 * Maintain the project by means of fixing bugs and providing support for new releases.
-* Ensure the good reputation of Ansys in the open source community.
+* Uphold Ansys' reputation in the open-source community.
```

Comment:
Thanks for the reminder @PipKat .

Reply:
@akaszynski  I didn't change it to Ansys's reputation--because all of our corporate/marketing stuff uses Ansys' rather than Ansys's. The rule here--as you and I know--is if you say the "s" when forming the possessive of a word ending in "s", then you add the "s". However, I figured I should follow what we do in our corporate communications.